omod/src/main/webapp/resources/htmlforms/Lab.xml

```diff
@@ -3025,12 +3025,22 @@
 	<br/>
 	<br/>
 	<div id="datePr">
-	Date de Prochaine Visite :<obs conceptId="CIEL:5096" allowFutureDates="true"/><br/><br/>
-	Nom d'auteur de la fiche :<encounterProviderAndRole encounterRole="322fc5ba-2ed6-44cf-8ebc-963dc40322af" autocompleteProvider="true" default="currentUser"/> <br/><br/>
+		Date de Prochaine Visite :<obs conceptId="CIEL:5096" allowFutureDates="true"/><br/><br/>
+		Nom d'auteur de la fiche :<encounterProviderAndRole encounterRole="322fc5ba-2ed6-44cf-8ebc-963dc40322af" autocompleteProvider="true" default="currentUser"/> <br/>
+
+		Envoyez le résultat de laboratoire à: <br/>
+		<input id="send-lab-order-to-SCC" type="checkbox" />
```

Comment:
It is required to add the name attribute in order to make it possible to fetch this data on the backend.

Reply:
outdated comment. Now, we pass order destination by a a generic text concept